src/clusterfuzz/_internal/platforms/android/util.py

```diff
@@ -37,3 +37,43 @@ def get_local_path(device_path):
   return os.path.join(
       root_directory,
       os.path.relpath(device_path, android.constants.DEVICE_FUZZING_DIR))
+
+
+def is_testcase_deprecated(platform_id=None):
+  """Whether or not the Android device is deprecated."""
+
+  # Platform ID for Android is of the form - |android:{codename}_{sanitizer}:{build_version}|
+  platform_id_fields = platform_id.split(':')
```

Comment:
hmmm...I'm not so happy with this string manipulation. seems fragile. WDYT @oliverchang

Reply:
Friendly ping on this @oliverchang